packages/preview/sdu-lab-report/0.1.0/typst.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "sdu-lab-report"
+version = "0.1.0"
+entrypoint = "template.typ"
+authors = ["Arshtyi"]
+license = "MIT"
+description = "A SDU lab report."
+keywords = ["lab", "report", "sdu"]
+categories = ["report"]
+repository = "https://github.com/Arshtyi/SDU-Lab-Report"
+compiler = "0.14.1"
+exclude = ["sdu.png"]
+
+[template]
+path = "example"
+entrypoint = "main.typ"
+thumbnail = "example/figures/main.png"
```

Comment:
The thumbnail shouldn't be inside the template directory, since it all files will be copied when a user intializes this template.